config/prow/plugins.yaml

```diff
@@ -99,6 +99,10 @@ approve:
   - helm/charts
   require_self_approval: false
   lgtm_acts_as_approve: true
+- repos:
+  - kubernetes/cloud-provider-openstack
```

Comment:
you can add this line after 99 before 100 i think. since that section has the flags you need ( require_self_approval - false and ignore_review_state - true)

Reply:
it doesn't include `ignore_review_state`, does it?

Reply:
whoops. you are right